servicecontrol/creating-monitoring-config-file.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Monitoring Configuration Settings
```

Comment:
> Monitoring Instance Configuration Settings

Without the "Instance" it feels like we're saying this is how you monitor config settings